evm/db/backends/level.py

```diff
@@ -0,0 +1,48 @@
+import shutil
+from .snapshot import Snapshot
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+
+    # Creates db as a class variable to avoid level db lock error
+    def __init__(self):
+        try:
+            import leveldb
+        except ImportError:
+            raise ImportError("LevelDB requires the leveldb \
+                               library which is not available for import.")
+        if not LevelDB.db:
+            LevelDB.db = leveldb.LevelDB('./evm/db/backends/db',
```

Comment:
Just a note that this should be configurable at some point but it's not important immediately.